src/pages/en/tutorial/2-pages/5.mdx

```diff
@@ -108,19 +109,27 @@ Your About page is now styled using *both* the imported `global.css` file *and*
 
 - Are styles from both styling methods being applied? 
 
-    || Yes ||
+    <p>
+      <Spoiler>Yes</Spoiler>
+    </p>
 
 - Are there any conflicting styles, and if so, which are applied?
 
-    || Yes, h1 has a size of 2.5rem globally, but 4rem locally in the <style> tag. The local 4rem rule is applied on the About page. ||
+    <p>
+      <Spoiler>Yes, h1 has a size of 2.5rem globally, but 4rem locally in the `<style>` tag. The local 4rem rule is applied on the About page.</Spoiler>
```

Comment:
```suggestion
      <Spoiler>Yes, `<h1>`  has a size of `2.5rem` globally, but `4rem` locally in the `<style>` tag. The local `4rem` rule is applied on the About page.</Spoiler>
```
FWIW, this is what we had before. I think making h1 a tag makes sense, and open to offers on whether we want the `rem` units back ticked!

Reply:
Oh yes, should definitely add these back 👍